nntrainer/src/neuralnet.cpp

```diff
@@ -918,7 +1016,7 @@ int NeuralNetwork::train_run() {
           return status;
         }
         std::cout << "#" << i + 1 << "/" << epoch;
-        data_buffer->displayProgress(iter, nntrainer::BUF_TRAIN, getLoss());
+        data_buffer->displayProgress(count++, nntrainer::BUF_TRAIN, getLoss());
```

Comment:
Can you elaborate on the usage of `count` as I think `iter` does the same job.
`iter` tracks class state and can be stored to continue the training.

Reply:
This is show how many data is calculated in every epoch.

Reply:
this counter is used for counting how many data is calculated in every epoch. However, iter is used to update decay.

Reply:
Got it.